RiotSwiftUI/Modules/Onboarding/SplashScreen/View/OnboardingSplashScreenPage.swift

```diff
@@ -0,0 +1,89 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+@available(iOS 14.0, *)
+struct OnboardingSplashScreenPage: View {
+    
+    // MARK: - Properties
+    
+    // MARK: Private
+    @Environment(\.theme) private var theme
+    
+    // MARK: Public
+    /// The content that this page should display.
+    let content: OnboardingSplashScreenPageContent
+    /// The height of the non-scrollable content in the splash screen.
+    let overlayHeight: CGFloat
+    
+    var isDarkModeEnabled: Bool {
```

Comment:
This is definitely something we should add on theme protocol. Like `isDark`, it may be confusing with the DarkTheme. But just a property to advertise if the theme tend to be dark can be helpful to avoid doing the same checks in every places.